charger/ocpp/instance.go

```diff
@@ -16,11 +16,37 @@ import (
 	"github.com/lorenzodonini/ocpp-go/ws"
 )
 
+type Config struct {
+	Port        int    `json:"port"`
+	ExternalUrl string `json:"externalUrl,omitempty" yaml:",omitempty"`
```

Comment:
Das sollte es nach den letzten Netzwerkanpassungen nicht geben.

Reply:
> When running trough a reverse proxy this setting is not equal to network.externalurl

The differences are schema and port, nothing else. This should not be a separate configuratoin.

Reply:
> parse 

Would actually make sense to make user externalurl is really parsable if manally set and make it an `url.URL`.

Reply:
Less explicit, more magic, but would be fine for me. I've added a task.

Reply:
Why is the Fallback unwanted?